dev-client/src/screens/CreateProjectScreen/components/CreateProjectForm.tsx

```diff
@@ -98,7 +111,7 @@ const FormContainer = React.memo(
         <Box position="absolute" bottom={8} right={3} p={3}>
           <Button
             onPress={handleSubmit}
-            disabled={isSubmitting}
+            isDisabled={isSubmitting || !isValid} //TODO-cknipe: Comment in PR: is this what was intended with isSubmitting?
```

Comment:
I'm assuming the new behavior is what was intended originally but I don't think it worked, because when I put the isValid check under `disabled`, it didn't actually appear to disable the button when invalid.